node/overseer/overseer-gen/proc-macro/src/parse_struct.rs

```diff
@@ -280,15 +281,6 @@ impl OverseerInfo {
 			.collect::<Vec<TokenStream>>()
 	}
 
-	/// Generic types per subsystem, as defined by the user.
-	pub(crate) fn builder_generic_types(&self) -> Vec<Ident> {
-		self.subsystems
-			.iter()
-			.filter(|ssf| !ssf.wip)
-			.map(|sff| sff.generic.clone())
-			.collect::<Vec<_>>()
-	}
```

Comment:
I actually think that this level does not necessarily have to be changed. Happy to be convinced otherwise.

Reply:
It should actually be `subsystem_generic_types` then, not `builder_generic_types` as it covers merely subsystem and missing potential generics from baggage (and we have `baggage_generic_types` as well).

Reply:
Done in 5d0503c07fa70d6f19e8d754a5e2f17b165b0150